tdrs-backend/tdpservice/users/migrations/0012_add_initial_super_user.py

```diff
@@ -0,0 +1,26 @@
+import os
+from django.db import migrations
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0011_auto_20210108_1741'),
+    ]
+
+    def generate_superuser(apps, schema_editor):
+        from ..models import User
+
+        # DJANGO_DB_NAME = os.environ.get('DJANGO_DB_NAME', "default")
+        DJANGO_SU_NAME = os.environ.get('DJANGO_SU_NAME','admin')
+        DJANGO_SU_EMAIL = os.environ.get('DJANGO_SU_EMAIL','admin@example.com')
+        DJANGO_SU_PASSWORD = os.environ.get('DJANGO_SU_PASSWORD','StrongPassword') # only use the default case for password in development
```

Comment:
We should only have to set the username here, as the email isn't required initially and the password is set to unusable. We should also update the docs to instruct developers to add these environment variables and update them with their own info. We should also update `.env.example`